workspaces/rstudio/Dockerfile

```
@@ -47,63 +47,77 @@
+chown -R 1001:1001 /var/lib/rstudio-server
+chown -R 1001:1001 /var/run/rstudio-server
+chown -R 1001:1001 /etc/rstudio
+
 # We store some generated files and the Nginx pid file in /var/pl-var.
-RUN mkdir -p /var/pl-var && \
-    chown -R 1001:1001 /var/lib/nginx /var/log/nginx /var/pl-var
+mkdir -p /var/pl-var
+chown -R 1001:1001 /var/lib/nginx /var/log/nginx /var/pl-var
+
+EOF
 # The Nginx proxy allows us to change the base URL for the RStudio session,
```

Hadolint (2.12.0)

[error] unexpected '#'
expecting a new line followed by the next instruction

(DL1000)